src/Discord.Net.Core/Entities/Integrations/IIntegration.cs

```diff
@@ -3,15 +3,16 @@
 namespace Discord
 {
     /// <summary>
-    ///     Holds information for a guild integration feature.
+    ///     Holds information for an integration feature.
+    ///     Nullable fields not provided for Discord bot integrations, but are for Twitch etc.
```

Comment:
To prevent some headaches with nullables maybe we could make an interface system under this with the correct properties that are non null, ex:
```md
IIntegration
- IDiscordBotIntegration
- ITwitchIntegration
- etc.
```